x/adminmodule/abci.go

```diff
@@ -1,71 +1,15 @@
 package adminmodule
 
 import (
-	"fmt"
 	"time"
 
 	"github.com/cosmos/admin-module/x/adminmodule/keeper"
 	"github.com/cosmos/admin-module/x/adminmodule/types"
 	"github.com/cosmos/cosmos-sdk/telemetry"
 	sdk "github.com/cosmos/cosmos-sdk/types"
-	govtypes "github.com/cosmos/cosmos-sdk/x/gov/types"
-	govv1beta1types "github.com/cosmos/cosmos-sdk/x/gov/types/v1beta1"
 )
 
-// EndBlocker called every block, process inflation, update validator set.
-func EndBlocker(ctx sdk.Context, keeper keeper.Keeper) {
+// EndBlocker called every block
+func EndBlocker(_ sdk.Context, _ keeper.Keeper) {
```

Comment:
I think we're safe to remove this func now when we removed all the EndBlock proposals processing logic